go/app/server_test.go

```diff
@@ -1,14 +1,21 @@
 package app
 
 import (
+	"bytes"
+	"database/sql"
+	"errors"
+	"fmt"
+	"mime/multipart"
 	"net/http"
 	"net/http/httptest"
-	"net/url"
+	"os"
+	"path/filepath"
 	"strings"
 	"testing"
 
+	"github.com/golang/mock/gomock"
```

Comment:
[must]
これは `"go.uber.org/mock/gomock"` に置き換えてください！

https://github.com/golang/mock の README にも書かれている通り、このツールはメンテナンスされていないので `go.uber.org/mock` を利用するように書かれています。
```suggestion
	"go.uber.org/mock/gomock"
```

> Update, June 2023: This repo and tool are no longer maintained. Please see [go.uber.org/mock](https://github.com/uber/mock) for a maintained fork instead.